bundles/org.openhab.binding.xmltv/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -4,6 +4,5 @@
 	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
 
 	<name>XmlTV Binding</name>
-	<description>This is the binding for reading and parsing XmlTV files</description>
```

Comment:
Why do you suppress the description ?

Reply:
Descriptions are superfluous and redundant as soon as they are present in the properties file.

Reply:
@cweitkamp @wborn : what is your opinion about removing all description tags from XML files?

Reply:
I don't have any plans to do the same for the add-ons I maintain. It takes effort and if the descriptions are good, they provide some context to things/channels that usually only have very short labels. So IMHO the descriptions help with understanding how add-ons are modeled.

Reply:
That is also my opinion.

Reply:
Don't get me wrong : removing description does not impact their presentation as soon as they are present in the properties file : 
![image](https://user-images.githubusercontent.com/3328789/142718964-720f3858-92df-4d81-b944-dfea92abd771.png)

I was surprised by this when I discovered it and thought it would avoid having two places to maintain the same string.

Reply:
In this case, the description in the XML is more the original vision of the author and it is easy for any code maintainer to find it in the main place with everything else from the thing, that is the XML file.

Reply:
Ok, I understand, but should I also reintroduce descriptions on things, channels ... ?

Reply:
Yes ;)

Reply:
Here you are.